From d479458464b62b3772fc0d3b2a4077a0cddb2ec6 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Tue, 1 Oct 2024 17:42:49 +0200
Subject: [PATCH 1262/1695] dts: stm32: remove export vddgpu on stm32mp23

Remove the export of vddgpu as this is now managed
by a power domain.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e4afbd099121bf5530347916deef2177e12887f5 dts: stm32: remove export vddgpu on stm32mp23]
MP25 sync with : https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/401353
Change-Id: I31785761753856a86c3b19932715f51c1521a0a1
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/408142
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp231.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 98712cf2b..02dafae78 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -1164,11 +1164,6 @@
 						reg = <VOLTD_SCMI_ADC>;
 						voltd-supply = <&vdda18adc>;
 					};
-
-					voltd-vddgpu {
-						reg = <VOLTD_SCMI_GPU>;
-						voltd-supply = <&vddgpu>;
-					};
 				};
 			};
 		};
-- 
2.47.2

